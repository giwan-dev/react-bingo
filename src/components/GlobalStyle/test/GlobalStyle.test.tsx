import React from 'react';
import { shallow } from 'enzyme';
import GlobalStyle from '..';

it('should render without crashing', () => {
  const style = shallow((
    <GlobalStyle />
  ));
  expect(style).toMatchSnapshot();
});
