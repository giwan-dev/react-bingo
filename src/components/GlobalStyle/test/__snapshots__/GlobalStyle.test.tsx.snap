// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
